test/ts-scenario/steps/lib/contract.ts

```diff
@@ -4,16 +4,16 @@
 
 'use strict';
 
-import { Constants } from '../constants';
+import {Constants} from '../constants';
 import * as BaseUtils from './utility/baseUtils';
-import { CommandRunner } from './utility/commandRunner';
+import {CommandRunner} from './utility/commandRunner';
 
 import * as path from 'path';
-import { CommonConnectionProfileHelper } from './utility/commonConnectionProfileHelper';
 
 const commandRunner: CommandRunner = CommandRunner.getInstance();
 
 // CLI verbosity in commands
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
See comment for same code in `channel.ts`. Since it's used more than once, maybe there should be a helper function to get the boolean value, used both places